packages/mobile/src/screens/app-screen/useAppScreenOptions.tsx

```diff
@@ -95,18 +97,18 @@ export const useAppScreenOptions = (
   }, [navigation])
 
   const { isEnabled: isEarlyAccess } = useFeatureFlag(FeatureFlags.EARLY_ACCESS)
+  const drawerProgress = useDrawerProgress()
 
   const screenOptions: (options: Options) => NativeStackNavigationOptions =
     useCallback(
       (options) => {
-        const { navigation } = options
-        // The manual typing is unfortunate here. There may be a better way, but
-        // the tricky bit is that StackNavigationOptions aren't known to the RouteProp.
-        // A better solution may be to wrap <Stack.Screen> in our own variant that
-        // can do some better generics & inference.
+        const { navigation, route } = options
+        const { params } = route
+        const isFromAppLeftDrawer =
+          params && (params as ContextualParams).fromAppDrawer
 
         return {
-          animation: 'default',
+          animation: isFromAppLeftDrawer ? 'none' : 'default',
```

Comment:
Nice. good solution. im thinking we should do something more than `default` at some point. It's kinda slower than I'd like and I think we can add some good physics.